src/common/tools.ts

```diff
@@ -254,3 +254,11 @@ export const waitAMoment = (ms = 100): Promise<boolean> => {
 }
 
 export const numToFixed = (number: number, digits = 3): number => parseFloat(number.toFixed(digits))
+
+export const jumpToErrorFormItem = (className = '.el-form-item.is-error'): void => {
+  const el = document.querySelector(className)
+  if (el) {
+    el.scrollIntoView()
```

Comment:
```suggestion
    el.scrollIntoView({behavior: "smooth"})
```

Using the behavior attribute allows for a smoother transition to scrolling, which should not look too abrupt. 🤔